### DISCUSSION TITLE
media.macosicons.com returning Error 522

### DISCUSSION THREAD

**Discussion by tsuliwaensis** [elrumo/macOS_Big_Sur_icons_replacements#1281] [Help with the website]
And the main website is not loading category icons (Popular, Extensions, etc). 

Not sure if it's related to the 522 but inspector is returning 

> Access to XMLhttpRequest at https://media.macosicons.com/parse/classes/Icons2 from origin https://macosicons.com has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No Access-Control-Allow-Origin header is present on the requested resource

**Comment by alexfazio** [comment]
Website still broken today :(

**Comment by traaaaaaainnnnnnnnnns** [comment]
@elrumo I'm getting a 502 bad gateway error on Cloudflare

**Comment by scris** [comment]
I got the same error downloading icons

**Comment by isametry** [comment]
Seeing a

```
404 Not Found
---
nginx/1.18.0 (Ubuntu)
```

on a lot of the icons.

**Comment by alexfazio** [comment]
@elrumo are you having hosting issues? Please let us know if we can help

**Comment by elrumo** [comment]
Hey everyone! There were some issues with the hosting provider so the server was down for a few hours, yesterday it finally went back up, but something must have changed to the network because I can't seem to start the MongoDB where all the icons are stored. I'm trying to work it out. 

@alexfazio If you have any experience with MongoDB let me know, as that could be very helpful.

PS. This happened on my first day of holiday, so I might be a bit slow to respond...

**Reply by alexfazio** [reply]
Thanks for the update, I use your icon lib often and was afraid of seeing it die! Let us know how it goes

**Comment by bastinald** [comment]
Can you post the mongoDB errors you get when booting the server? I can tell you how to fix it.

**Reply by alexfazio** [reply]
@elrumo

**Comment by rmonvfer** [comment]
Hi! I have some experience managing MongoDB clusters so I might be able to help.
Could you provide the MongoDB log file? (it should be at `/var/log/mongodb/mongodb.log` if you haven't modified the config file)

**Comment by andylin2004** [comment]
Now the server returns an error 524
